src/common/framework/fixture.ts

```diff
@@ -237,16 +237,26 @@ export class Fixture<S extends SubcaseBatchState = SubcaseBatchState> {
   }
 
   /** Expect that the provided promise rejects, with the provided exception name. */
-  shouldReject(expectedName: string, p: Promise<unknown>, msg?: string): void {
+  shouldReject(
+    expectedName: string,
+    p: Promise<unknown>,
+    { checkForStackProperty = false, message }: ExceptionCheckOptions = {}
```

Comment:
General question, should the default use case actually be `checkForStackProperty = true` instead? And then we add exceptions to that case when relevant? (I was under the impression that by default we wanted to check that, and also seems like a lot of the test changes are related to overriding the default value?)